test/integration/sonobuoy_integration_test.go

```diff
@@ -513,18 +512,15 @@ func TestExactOutput(t *testing.T) {
 			desc:       "gen with config then flags targeting subfields",
 			cmdLine:    "gen --config=testdata/subfieldTest.json -n cmdlineNS --image-pull-policy=Always --sonobuoy-image=cmdlineimg --timeout=99",
 			expectFile: "testdata/gen-config-then-flags.golden",
-		}, {
-			desc:       "gen with flags targeting subfields then config",
-			cmdLine:    "gen -n=cmdlineNS --image-pull-policy=Always --sonobuoy-image=cmdlineimg --timeout=99 --config=testdata/subfieldTest.json",
-			expectFile: "testdata/gen-flags-then-config.golden",
 		},
 	}
 	for _, tc := range testCases {
 		t.Run(tc.desc, func(t *testing.T) {
-			output := mustRunSonobuoyCommandWithContext(ctx, t, tc.cmdLine)
+			// Allow errors here since we also may test stderr
+			output, _ := runSonobuoyCommandWithContext(ctx, t, tc.cmdLine)
+
 			binaryVersion := mustRunSonobuoyCommand(t, "version --short")
 			binaryVer := strings.TrimSpace(binaryVersion.String())
-			t.Logf("version output of binary, after trimming space: %q\n", binaryVer)
```

Comment:
Just needed this for a while; after looking at more logs it isnt quite necessary.